apps/web/src/server/lib/express-validation-errors.js

```diff
@@ -12,7 +12,7 @@ export const expressValidationErrorsInterceptor = (request, response, next) => {
 	const errors = validationResult(request);
 
 	if (!errors.isEmpty()) {
-		request.validationErrors = errors.mapped();
+		response.locals.errors = errors.mapped();
 
 		request.body.errors = errors.mapped();
```

Comment:
We should refactor and deprecate the `errors` and `errorSummary` from the body and move into locals right?

Please add a TODO here so I can pick it up and refactor once this is merged.

Reply:
Sure, though I've been begun doing this as part of the refactoring we mentioned yesterday afternoon. Should I continue with it, or do you want to?

Reply:
(As discussed in morning meeting, I'll pick this up)